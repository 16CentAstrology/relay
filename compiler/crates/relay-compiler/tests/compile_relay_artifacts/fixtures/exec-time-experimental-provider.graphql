query execTimeExperimentalProviderQuery @exec_time_resolvers(enabledProvider: "execTimeResolversFlagProvider", useExperimentalProvider: "useExperimentalProvider") {
  me {
    name
  }
}
