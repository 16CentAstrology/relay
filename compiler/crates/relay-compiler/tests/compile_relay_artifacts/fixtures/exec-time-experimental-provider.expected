==================================== INPUT ====================================
query execTimeExperimentalProviderQuery @exec_time_resolvers(enabledProvider: "execTimeResolversFlagProvider", useExperimentalProvider: "useExperimentalProvider") {
  me {
    name
  }
}
==================================== OUTPUT ===================================
{
  "fragment": {
    "argumentDefinitions": [],
    "kind": "Fragment",
    "metadata": null,
    "name": "execTimeExperimentalProviderQuery",
    "selections": [
      {
        "alias": null,
        "args": null,
        "concreteType": "User",
        "kind": "LinkedField",
        "name": "me",
        "plural": false,
        "selections": [
          {
            "alias": null,
            "args": null,
            "kind": "ScalarField",
            "name": "name",
            "storageKey": null
          }
        ],
        "storageKey": null
      }
    ],
    "type": "Query",
    "abstractKey": null
  },
  "kind": "Request",
  "operation": {
    "argumentDefinitions": [],
    "kind": "Operation",
    "name": "execTimeExperimentalProviderQuery",
    "selections": [
      {
        "alias": null,
        "args": null,
        "concreteType": "User",
        "kind": "LinkedField",
        "name": "me",
        "plural": false,
        "selections": [
          {
            "alias": null,
            "args": null,
            "kind": "ScalarField",
            "name": "name",
            "storageKey": null
          },
          {
            "alias": null,
            "args": null,
            "kind": "ScalarField",
            "name": "id",
            "storageKey": null
          }
        ],
        "storageKey": null
      }
    ],
    "exec_time_resolvers_enabled_provider": require('execTimeResolversFlagProvider'),
    "use_experimental_provider": require('useExperimentalProvider')
  },
  "params": {
    "cacheID": "654ebcae93bbbc912b6510d6732c9e4b",
    "id": null,
    "metadata": {},
    "name": "execTimeExperimentalProviderQuery",
    "operationKind": "query",
    "text": null
  }
}

QUERY:

query execTimeExperimentalProviderQuery {
  me {
    name
    id
  }
}
